pkg/release/release.go

```diff
@@ -19,17 +19,17 @@ package release
 var Base = "4.10.18"
 
 var Image = map[string]string{
-	"cli":                     "quay.io/microshift/cli:" + Base,
-	"coredns":                 "quay.io/microshift/coredns:" + Base,
-	"haproxy_router":          "quay.io/microshift/haproxy-router:" + Base,
-	"kube_rbac_proxy":         "quay.io/microshift/kube-rbac-proxy:" + Base,
-	"pause":                   "quay.io/microshift/pause:" + Base,
-	"service_ca_operator":     "quay.io/microshift/service-ca-operator:" + Base,
-	"openssl":                 "quay.io/microshift/openssl" + Base,
-	"ovn_kubernetes":          "quay.io/microshift/ovn-kubernetes-singlenode:" + Base,
-	"odf_topolvm":             "quay.io/microshift/odf-topolvm-rhel8" + Base,
-	"ose_csi_ext_provisioner": "quay.io/microshift/ose-csi-external-provisioner" + Base,
-	"ose_csi_ext_resizer":     "quay.io/microshift/ose-csi-external-resizer" + Base,
-	"ose_csi_node_registrar":  "quay.io/microshift/ose-csi-node-driver-registrar" + Base,
-	"ose_csi_livenessprobe":   "quay.io/microshift/ose-csi-livenessprobe" + Base,
+	"cli":                       "quay.io/microshift/cli:" + Base,
+	"coredns":                   "quay.io/microshift/coredns:" + Base,
+	"haproxy_router":            "quay.io/microshift/haproxy-router:" + Base,
+	"kube_rbac_proxy":           "quay.io/microshift/kube-rbac-proxy:" + Base,
+	"pause":                     "quay.io/microshift/pause:" + Base,
+	"service_ca_operator":       "quay.io/microshift/service-ca-operator:" + Base,
+	"openssl":                   "quay.io/microshift/openssl" + Base,
+	"ovn_kubernetes-microshift": "quay.io/microshift/ovn-kubernetes-microshift:" + Base,
```

Comment:
You'll probably want to use underscores only (i.e. `ovn_kubernetes_microshift`). We're using this same name in the Go templating of the resource manifests and IIRC, the templating language stumbles over hyphens.

BTW: You'd need to update your manifests in this same PR.

Reply:
Fixed the image name.
@fzdarsky Do you mean I need to update the image name in ovnk manifest?  The image name in ovnk manifests are rendered from config parameters. Other parts of the ovnk manifests are up-to-date I think.

Reply:
> Fixed the image name. @fzdarsky Do you mean I need to update the image name in ovnk manifest? The image name in ovnk manifests are rendered from config parameters. Other parts of the ovnk manifests are up-to-date I think.

Thank you.

Yes, you need to update the manifests (https://github.com/openshift/microshift/blob/main/assets/components/ovn/master/daemonset.yaml#L47 et al.), because we're now adding the whole `release.Image` map to the rendering parameters by default (see https://github.com/openshift/microshift/blob/main/pkg/components/render.go#L16), so we no longer have to add them on a case-by-case basis. Result is the names need to match, though.

Reply:
I see, thanks!
I forgot to update the image var in manifest.
updated the image var to ovn_kubernetes_microshift